docs/additional-reading/upgrade-to-version-8-0-0.md

```diff
@@ -1,37 +1,190 @@
+
+    ```yml 
+    # Note: Base output directory of /public is assumed for static files
+    default: &default
+      manifest: manifest.json
+      # Used in your webpack configuration. Must be created in the
+      # webpack_public_output_dir folder
+    
+    development:
+      <<: *default
+      # generated files for development, in /public/webpack/development
+      webpack_public_output_dir: webpack/development
+    
+      # Default is localhost:3500
+      hot_reloading_host: localhost:3500
+    
+      # Developer note: considering removing this option so it can ONLY be turned by using an ENV value.
+      # Default is false, ENV 'HOT_RELOAD' will always override
+      hot_reloading_enabled_by_default: false
+    
+    test:
+      <<: *default
+      # generated files for tests, in /public/webpack/test
+      webpack_public_output_dir: webpack/test
+    
+    production:
+      <<: *default
+      # generated files for tests, in /public/webpack/production
+      webpack_public_output_dir: webpack/production
+    
+    ```
+
+4. You'll want to make sure you remove any references to your webpack bundles from any files in your asset pipeline. For example, a common reference you might have is a line in your `assets/javascripts/application.js` like the one shown below:
+ 
+    ```javascript
+     //= require webpack-bundle
+    ```
+
+5. You'll need to add the approrpriate webpacker_lite helpers to your layouts to load the files from  your public deployment directory. The helpers ensure that the bundle names are converted to the correct paths on your rails server. 
+
+   For example, you can do this by adding something like this to your `application.html.erb`:
+   ```erb
+   <%= javascript_pack_tag 'main' %>
+   ```
+
+6. You'll need to configure your webpack-bundle files so that they are placed into your public directory and skip the asset pipeline. For example, you may have some javascript file in your client directory called something like `webpack.config.js`. This webpack config file should have some output code that dictates what path your webpack-bundles are placed into. For example, a change to your code might look something like this:
+
+    ```javascript
+    // client/webpack.config.js 
+    // before
+    output: {
+      filename: 'webpack-bundle.js',
+      path: pathLib.resolve(__dirname, '../app/assets/webpack'),
+    },
+    ```
+    
+    ```javascript
+    // client/webpack.config.js
+    // after
+    output: {
+      filename: 'webpack-bundle.js',
+      path: pathLib.resolve(__dirname, '../app/public/webpack'),
+    },
+    ```
+    
+7. You'll also need to configure webpack to use your manifest appropriately. You'll need to configure webpack to knows your manifest's location and so on. 
```

Comment:
@justin808 Everything from here-on is still WIP. Generally, we need to explain how to configure webpack using react_on_rails's handy webpackConfigLoader